core/src/test/scala/integration/kafka/api/DescribeAuthorizedOperationsTest.scala

```diff
@@ -32,17 +32,55 @@ import org.junit.jupiter.api.{AfterEach, BeforeEach, Test}
 
 import scala.jdk.CollectionConverters._
 
+object DescribeAuthorizedOperationsTest {
+  val Group1 = "group1"
```

Comment:
Is ```GROUP_1" more better? it is global variable in this test.

Reply:
In Scala, we tend to only capitalize the first letter of constants. At least, this how I have been doing it so far. I prefer to keep it as is.